.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
How about renaming this file to `.prettierrc.json` to get the correct syntax highlighting for free?